packages/react-components/react-combobox/library/src/components/Listbox/useListbox.ts

```diff
@@ -178,6 +178,7 @@ export const useListbox_unstable = (props: ListboxProps, ref: React.Ref<HTMLElem
     };
 
     // this should only be run once in the lifecycle of the Listbox
+    // eslint-disable-next-line react-compiler/react-compiler
```

Comment:
The issue i unrelated to the PR changes; it's likely that the eslint configuration or dependencies were updated.

Reply:
it could be merge race condition, if you revert this change will the build break?

Reply:
This is fixed by #31795

For future reference we're using the `'use no memo'` directive as that instructs React Compiler to ignore this code and also disables the linter. This is nice because it better ensures no breakage for downstream Compiler users and gives us a nice, findable string for potential future improvements/fixes.